docs/user-guide/projects/project-settings/modules/README.md

```diff
@@ -12,6 +12,6 @@ To **activate or deactivate modules** in a project select a project from the sel
 
 Setting a check mark next to a project module will activate the module in the project and a new menu item will appear in the project menu on the left.
 
-![select-modules-OpenProject](image-20201005151029444.png)
+![image-20220207105240658](image-20220207105240658.png)
```

Comment:
hi @Fweber1331, please make sure to give the image descriptions in [...] a real name, e.g "[activate modules in OpenProject]" and also ideally the image itself, e.g. "(activate-modules.png)".

Reply:
@birthe Got it! Fixed in Calendar, Modules, Team Planner and Use Cases